sha-256 as MTI (#130)
oauthstuff/draft-selective-disclosure-jwt: main.md

identifiers not found in this registry are not considered valid and MUST NOT be
accepted by verifiers.

> To promote interoperability among implementations, the SHA-256 hash algorithm is mandatory to implement.

Reviewer: ... the SHA-256 hash algorithm MUST be implemented.

Reviewer 2: Agreed. Perhaps something even shorter like:

Implementations MUST support the SHA-256 hash algorithm as shown in Example 1.

Reviewer 2: Agreed with @peppelinux proposal. It may be sufficient to just say:

Implementations MUST support the SHA-256 hash algorithm ()

> To promote interoperability among implementations, the SHA-256 hash algorithm is mandatory to implement.

Author: ```suggestion
To promote interoperability, implementations MUST support the SHA-256 hash algorithm.
```
